### ISSUE TITLE
README.md should document .bowerrc and Bowerfile approaches separately

### ISSUE THREAD

**Issue by cjheath** [rharriso/bower-rails#158]
It seems that some features are available by using a Bowerfile, and some by using a .bowerrc, and some both ways. It's not at all clear which is which, or whether I can or must use both, and it's not clear when the two might conflict. If there are two (or more!) different approaches, please document them clearly and separately, so new users know what they have to use to get going.

**Comment by alexvbush** [comment]
Agree, looking at the current README I can barely understand how to get started.... I don't know why I need all these setup... I just want to get started with minimum default/standard setup to use a bower package or two in my rails app.

**Comment by alexvbush** [comment]
Something like this http://growingdevs.com/stop-using-rubygems-and-start-using-bower.html

**Comment by bc-shusson** [comment]
+1 and thanks for that link @alexvbush
